.eslintrc

```diff
@@ -6,7 +6,6 @@
     "@typescript-eslint/no-non-null-assertion": "off",
     "@typescript-eslint/no-explicit-any": "off",
     "@typescript-eslint/explicit-module-boundary-types": "off",
-    "@typescript-eslint/explicit-member-accessibility": "error",
```

Comment:
I guess this is removed by accident since it was just added in previous closed PR here? https://github.com/vaadin/fusion/pull/132/files#diff-e7e195d3d8fa5b82fa51ed952f3a179d429cbc46c0432ce1f7b357c54c162822R9

Reply:
No, I just found that it conflicts with the generator code (private class members). I would like to have it but until it doesn't handle private members correctly, we cannot use it (at least for generator code).